etc/kayobe/kolla.yml

```diff
@@ -12,6 +12,7 @@
 
 # URL of Kolla source code repository if type is 'source'.
 #kolla_source_url:
+kolla_source_url: https://git.openstack.org/openstack/kolla
```

Comment:
The default in kayobe is https://github.com/stackhpc/kolla, let's keep that.

Reply:
Do we need this source specifically for building magnum for P3?

Reply:
Yes - we don't have the stable/queens in our fork